INSTALL
// on main machine
$ python setup.py sdist 
// on PI
$ tar -xzf odol-0.22.tar.gz && cd odol-0.22/ && sudo python setup.py develop

$ /etc/rc.local
$ su pi -c '/usr/bin/python /usr/local/bin/odol-service.py -p /dev/ttyUSB0 > /home/pi/odol.log 2>&1 &' 
$ su pi -c '/usr/bin/python /usr/local/bin/odol-server.py > /home/pi/odol.log 2>&1 &'


TESTS
clear && python setup.py nosetests --nocapture -d

BUGS
history.txt wird nach test erzeugt und nicht gel√∂scht
www muss in package_data
tests/data fehlen
service startet nicht nach boot